### ISSUE TITLE
How can I add the text to pdf in the watermark way?

### ISSUE THREAD

**Issue by GITHUBblazer** [unidoc/unipdf-examples#252]
according to your watermark codes, found that place the image on the pdf is the most ways of achieving the watermark to the pdf,
I wonder whether it supports to use the text to watermark the pdf in a different way, 
eg. I give a number or letter only and .........

**Comment by ipod4g** [comment]
@GITHUBblazer thank you for reaching out
Would you mind elaborating further on what you're aiming to accomplish? 
Please provide us more details, example or send the expected PDF file would be really helpful and we'll try to assist you with your query.

**Comment by 3ace** [comment]
@GITHUBblazer yes, currently we haven't support adding text watermark directly. So one workaround you could use for now is to construct an image from text and then use that image as watermark image.

**Comment by anovik** [comment]
@GITHUBblazer Please check how to add text watermark https://github.com/unidoc/unipdf-examples/blob/master/text/pdf_add_text_watermark.go

AddWatermarkText() function was added in the latest release https://github.com/unidoc/unipdf/releases/tag/v3.66.0
